cmplibrary/src/main/java/com/sourcepoint/cmplibrary/creation/CreationalUtility.kt

```diff
@@ -34,13 +35,17 @@ internal fun createLogger(errorMessageManager: ErrorMessageManager): Logger {
     )
 }
 
-internal fun networkClient(netClient: OkHttpClient, responseManage: ResponseManager, logger: Logger): NetworkClient {
-    return createNetworkClient(
+internal fun networkClient(
```

Comment:
no need this change

Reply:
this is just a small refactor to fit to the Kotlin Code Styles, for example:
1. [Class headers﻿](https://kotlinlang.org/docs/coding-conventions.html#class-headers)
2. [Functions](https://kotlinlang.org/docs/coding-conventions.html#functions)
3. etc

Reply:
Sounds good but in a pr dedicated to a refactor, let’s keep the changes to the minimum in this pr